### ISSUE TITLE
Blocked policy - drive.google.com

### ISSUE THREAD

**Issue by berejant** [andrieslouw/imagesweserv#378]
Weserv blocks image hosted at Google Drive:

Example 1:
Origin URL: https://drive.google.com/uc?export=view&id=1fKbjdN_V6lLNTfY1uoA1oaLMSSQEN_a2
Weserv URL: https://images.weserv.nl/?url=https%3A%2F%2Fdrive.google.com%2Fuc%3Fexport%3Dview%26id%3D1fKbjdN_V6lLNTfY1uoA1oaLMSSQEN_a2

Example 2:
Origin url: https://drive.google.com/uc?export=view&id=1NA69YsCf1BhIXvGSr-RMB64DurPc5qAo 
Weserv url: https://images.weserv.nl/?url=https%3A%2F%2Fdrive.google.com%2Fuc%3Fexport%3Dview%26id%3D1NA69YsCf1BhIXvGSr-RMB64DurPc5qAo&w=238&h=163

Acutal error:
`{"status":"error","code":404,"message":"The hostname of the origin is unresolvable (DNS) or blocked by policy."}`

**Comment by andrieslouw** [comment]
Will need to investigate, as Google likes to block our servers/IP-ranges. See also #231.

**Comment by kleisauke** [comment]
It appears that Google Drive now sends a response header of significant size, exceeding our default 4k limit.
```console
$ curl --http1.1 -fsSL -o /dev/null -w "%{size_header}\n" 'https://drive.google.com/uc?export=view&id=1fKbjdN_V6lLNTfY1uoA1oaLMSSQEN_a2'
5604
```

This has been fixed with commit ca63088b93de7501eeb5d9f298380e36162b566f, which has just been rolled out to production. Thanks for reporting this!
